lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
> having the filter convert to a float just makes things less consistent. 

why is that? converting it to float will keep the same to_s behaviour as the other filters have (and all the other number filters also convert to float)